### ISSUE TITLE
EC:2.4.1.183	GO_REF:0000003

### ISSUE THREAD

**Issue by gocentral** [geneontology/go-ontology#10906]
Hey,

In pombe, ags1 is annotated to 
GO:0033201	alpha-1,4-glucan synthase activity IMP
GO:0047657	alpha-1,3-glucan synthase activity 	EC:2.4.1.183	GO_REF:0000003

I asked an author which one is correct (or both) and he replied that:

"The explanation is that they [the authors of the paper PMID:17472966] say that Ags1 is responsible for the accumulation of alpha1,4-glucan. It does not mean that Ags1 is the synthase that synthesizes directly this sugar, although it could be, and also it could not be. Nobody has demonstrated an enzymatic activity for Ags1, and nobody has been able to find an in vitro assay to measure alpha1,3 or alpha1,4-glucan synthase activity. So the only evidences are the sequence homology of Ags1 and that Ags1 overexpression produces an increase of sugars and Ags1 depletion causes a decrease of sugars.

In addition, the alpha-glucan of pombe seems to be a linear chain composed of a fragment of alpha1,4 followed by the rest of alpha1,3. The alpha1,4 is tiny fraction of 1/10 and the alpha1,3 is 9/10 of this chain.

When Ags1 overproduction o depletion is analyzed, it is not observed the increase or decrease of only alpha1,4, it is also observed a big change in the amount of alpha1,3-glucan.

In summary, Ags1 variation affects both alpha1,4 and alpha1,3-glucan, and it is responsible directly or indirectly for their synthesis.

Sometimes we refer to Ags1 and alpha-glucan without specifying 1,3 or 1,4, and sometimes we refer to 1,3 because it is 10-times more abundant than 1,4."

Are more things known in other organisms to make a specific annotation to alpha-1,3-glucan synthesis or should this IEA be alpha-glucan synthesis?

Cheers
Antonia. 

Reported by: antonialock

Original Ticket: [geneontology/ontology-requests/10720](https://sourceforge.net/p/geneontology/ontology-requests/10720)

**Comment by ukemi** [comment]
Added the following to GO:0033201
+xref: EC:2.4.1.21
+xref: RHEA:18192
Looks like the rxns are the same. Did not make a parent because the precise enzymatic activity of the gene product should be known for a function annotation. Otherwise an annotation to the process should be made.
